content/code/realtime/connection-recover.code

```diff
@@ -0,0 +1,95 @@
+[--- Javascript ---]
+var ably = new Ably.Realtime({
+      key: '{{API_KEY}}',
+      recover: function(_, cb) { cb(true); },
+      clientId: 'CHANGE-THIS-IF-YOU-WANT-MULTIPLE-CLIENTS'
+    }),
+    channel = ably.channels.get('recovery-presence'),
+    $present = $('#present');
+
+channel.presence.subscribe(function(message) {
+  channel.presence.get({}, function(err, members) {
+    var $list = $('<ul>');
+    members.forEach(function(member) {
+      var txt = "id: " + member.clientId + ", conn: " + member.connectionId;
+      $list.append($("<li>").text(txt));
+    });
+
+    if (members.length === 0) {
+      $present.html('<li>No one is present on this channel</li>');
+    } else {
+      $present.html($list.html());
+    }
+  });
+});
+
+$('#enter-presence').click(function() {
+  channel.presence.enter(function(err) {
+    var $actions = $('#actions');
+    if(err) {
+      $actions.append($("<li>").text(err));
+    } else {
+      $actions.append($("<li>").text('Entered as ' + ably.auth.clientId + ' with connection ID ' + ably.connection.id));
+    }
+  });
+});
+[--- /Javascript ---]
+
+[--- HTML ---]
+<html>
+<head>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+  <script src="//cdn.ably.io/lib/ably-1.js"></script>
+</head>
+<body>
+  <h1><a href="https://www.ably.io" target="_blank"><img src="/images/favicon.png">Ably connection recovery across reload</a> example</h1>
```

Comment:
I think it's worth linking this directly connection recovery docs instead of the homepage.